config/file.go

```diff
@@ -19,6 +19,7 @@ import (
 )
 
 var (
+	// ErrReadOnlyConfiguration is the error used for set configuration when it is read only.
```

Comment:
"used for set configuration" sounds a bit odd. Perhaps- "ErrReadOnlyConfiguration is returned when an attempt to modify a read-only configuration is made.".

Reply:
No problem, that sounded a bit odd indeed. I've just pushed the update.